tests/expose_test.go

```diff
@@ -388,7 +388,20 @@ var _ = Describe("Expose", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Waiting for the pod to report a successful connection attempt")
-				waitForJobToCompleteWithStatus(&virtClient, job, "success", 60)
+				waitForJobToCompleteWithStatus(&virtClient, job, "success", 120)
+
+				By("Starting an HTTP server on the VM, to verify HTTP connection also succeeds using the exposed VM.")
+				vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(vm.Name, &k8smetav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				tests.StartHTTPServer(vmi, testPort)
+
+				By("Starting a pod which tries to reach the VMI via ClusterIP over HTTP.")
+				job = tests.NewHelloWorldJobHttp(serviceIP, servicePort)
```

Comment:
If you really insist on checking `http` explicitly and not arbitrary `tcp`, then please change this test: https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_networking_test.go#L299 which checks cluster IP services with tcp traffic. However I don't see why `http` should be more important here than `tls`  or `ssh`, or why it should work differently. We are talking about cluster IP services and not about openshift routes which only work for `tls` and `http`.

Reply:
Ah ok, fine for me like it is. Still I don't see the value.